discover_overlay/voice_overlay.py

```diff
@@ -530,10 +530,10 @@ def draw_avatar(self, context, user, pos_x, pos_y, avatar_size):
                     avatar_size
                 )
         self.draw_avatar_pix(context, pix,pos_x,pos_y,colour, avatar_size)
-        if deaf:
-            self.draw_deaf(context, pos_x, pos_y, avatar_size)
-        elif mute:
-            self.draw_mute(context, pos_x, pos_y, avatar_size)
+        if mute:
+            self.draw_mute(context, pos_x, pos_y, bg_col, avatar_size)
```

Comment:
I think either we would need a new colour or to hardcode some semitransparent black here.

In my case bg_col here is solid black so it would simply blot out the image.

Reply:
isn't the bg_color at 50% of opacity ?

Reply:
This one's my fault. Colour system is REALLY badly named and documented.

bg_col in this instance is the colour used behind username text. It can be changed by user in settings but defaults to 50% transparent black. In my config it's 100% black and 100% green when they're talking

Reply:
hmm I see. 
indeed I didn't see this configuration